python/pyramid/audit/set-cookie-samesite.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+  - id: pyramid-set-cookie-samesite
+    patterns:
+      - pattern-either:
+          - pattern-inside: |
+              @pyramid.view.view_config(...)
+              def $VIEW($REQUEST):
+                  ...
+                  $RESPONSE = $REQUEST.response
+                  ...
+          - pattern-inside: |
+              def $VIEW(...):
+                  ...
+                  $RESPONSE = pyramid.httpexceptions.HTTPFound(...)
+                  ...
+      - pattern-not: $RESPONSE.set_cookie(..., samesite='Lax', ...)
+      - pattern-not: $RESPONSE.set_cookie(..., **$PARAMS)
+      - pattern: $RESPONSE.set_cookie(..., samesite=$SAMESITE, ...)
```

Comment:
It looks like this rule only checks for the case where `samesite` is set, but incorrectly. Is the default value secure?

Reply:
Fixed in 4ea7a800e8b951fceca2d6876a6e51c060a17fbe